e2e-tests/tfm/cypress/e2e/journeys/case/deal-AIN.spec.js

```diff
@@ -3,7 +3,7 @@ import relative from '../../relativeURL';
 import pages from '../../pages';
 import partials from '../../partials';
 import MOCK_DEAL_AIN from '../../../fixtures/deal-AIN';
-import { T1_USER_1, BANK1_MAKER1, ADMIN } from '../../../../../e2e-fixtures';
+import { ADMIN, BANK1_MAKER1, T1_USER_1 } from '../../../../../e2e-fixtures';
```

Comment:
How come these have swapped? Can leave just curious

Reply:
That's my bad I was playing around with my IDE linter to unintended consequences.. have reverted that back now!

Reply:
assuming done